kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/MainStream.java

```diff
@@ -0,0 +1,17 @@
+package com.kodilla.good.patterns.challenges;
+
+import java.util.stream.Collectors;
+
+public class MainStream {
+    public static void main(String[] args){
+
+        MovieStore movieStore = new MovieStore();
+
+         String theResult = movieStore.getMovies().entrySet().stream()
+                 .map(s -> s.getValue())
+                 .collect(Collectors.joining("!"));
```

Comment:
metoda
`Map<String, List<String>> getMovies`
zwraca Mape, gdzie key to String, value to Lista Stringow
movie.getMovies.entrySet() zwraca `Map.Entry<String, List<String>>`
s.getValue zwraca `List<String>`

Reply:
elementem streamu jest teraz List<String>. Musisz przeksztalcic je na stream String'ow. Dopiero wtedy bedziesz mogla uzyc `Collector.joining("!")`